### ISSUE TITLE
tslint.json of backbone-relational disables lots of rules

### ISSUE THREAD

**Issue by jgonggrijp** [DefinitelyTyped/DefinitelyTyped#32902]
According to the README, all lint rules should be enabled, but it looks like most or all of them are *dis*abled in backbone-relational:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/510f81374c99b2b985400faa92af10444c3b8127/types/backbone-relational/tslint.json#L3-L78

I noticed this while working on #32883. To be honest, this doesn't prevent me from using the package and I don't really care about it. However, I thought I should mention it anyway. It might be the case that more packages "suffer" from having lots of rules disabled, I didn't check for this.

~If you know how to fix the issue, make a pull request instead.~ Theoretically, I could do it, but I'm not willing to invest the potentially large amount of time required to get the module to pass all those rules.

- [ ] I tried using the `@types/backbone-relational` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @eirikhm 

~If you do not mention the authors the issue will be ignored.~

**Comment by SimonSchick** [comment]
tl;dr this was done to *many* packages when lint rules were beefed up, the maintainers of DT don't have the time lint each module by hand and it was more practical to just disable these rules.

You're welcome to make a PR.
